scripts/HFIR_4Circle_Reduction/absorption.py

```diff
@@ -0,0 +1,363 @@
+import numpy
+import numpy.linalg
+import math
+from mantid.api import AnalysisDataService
+
+# Do absorption correction
+
+
+class Lattice(object):
```

Comment:
Why do you implement your own Lattice class, why not use Mantids? `from mantid.geometry import UnitCell` It appears to have everything in it you require, see [here](http://docs.mantidproject.org/nightly/concepts/Lattice.html#unit-cells).